coriolis_openstack_utils/resource_utils/networks.py

```diff
@@ -109,3 +115,17 @@ def check_network_similarity(
         conflict_keys.add('subnets')
 
     return src_relevant_keys == conflict_keys
+
+
+def delete(openstack_client, tenant_id, network_name):
```

Comment:
Even if the parent module is named `networks` please always give explicit names to all functions (ex: `def delete_network(...):`)